fastlane/lib/assets/completions/completion.bash

```diff
@@ -0,0 +1,16 @@
+_fastlane_complete() {
+  COMPREPLY=()
+  local word="${COMP_WORDS[COMP_CWORD]}"
+  local completions=""
+
+  # look for Fastfile either in this directory or fastlane/ then grab the lane names
+  if [[ -e "Fastfile" ]]; then
+    completions=`cat Fastfile | grep "lane \:" | awk -F ':' '{print $2}' | awk -F ' ' '{print $1}'`
+  elif [[ -e "fastlane/Fastfile" ]]; then
+    completions=`cat fastlane/Fastfile | grep "lane \:" | awk -F ':' '{print $2}' | awk -F ' ' '{print $1}'`
+  fi
+
+  COMPREPLY=( $(compgen -W "$completions" -- "$word") )
```

Comment:
How feasible would it be to add this same magic for the commands.

e.g. something like

```
local gem_path = `gem which $tool`
completions = `cat $local_gem_path/lib/commands_generator | grep "command:" | blah blah` 
```